playbooks/openstack-osh_hostconfig.yml

```diff
@@ -19,19 +20,11 @@
         - SUSE-CA
         - OpenStack-Cloud8-product
         - SLE12Containers
-      tags:
-        - skip_ansible_lint
 
-    # TODO(evrardjp): Revert back to zypper module when dist-upgrade is fixed there
-    - name: Refresh zypper. After this step, we can rely on ansible modules.
-      raw: |
-        #TODO(evrardjp): Check if utilities/devel tools/virt/cloud8updates are really needed
-        #zypper ar -f -G https://provo-mirror.opensuse.org/repositories/utilities/SLE_12/ SLE12-utilities;
-        #zypper ar -f -G https://provo-mirror.opensuse.org/repositories/Virtualization:/containers/SLE_12_SP3/ Virtualization_containers;
-        #zypper ar -f -G  http://kirk.provo.novell.com/repositories/SUSE/Products/OpenStack-Cloud/8/x86_64/product/ openstackcloud-8-update
-        zypper refresh && zypper dist-upgrade -y
-      tags:
-        - skip_ansible_lint
+    - name: Zypper dist-upgrade
+      zypper:
+        name: '*'
+        state: dist-upgrade
```

Comment:
a zypper dist-upgrade on my machine doesn't actually dist-upgrade.
It runs and do nothing (just run dist-upgrade on the machine later on a machine that has tons of packages to update, and you'll see tons of them).

Is that me, or is it a bug with 12.3?

I moved to re-use zypper refresh && zyper dist-upgrade -y in raw due to that.

Reply:
This worked correctly on my machine (which is tumbleweed and has ton of packages to update daily)

could it be an ansible version issue? I wuld find that weird as it was introduced in 2017 and it is in versions 2.4 to 2.7 :thinking: 

Let me retest locally

Reply:
Could we try to remove that whole task, see if things fail anyway? I have the impression salt is doing the dist-upgrade anyway.